src/versionbits.cpp

```diff
@@ -194,15 +194,15 @@ class VersionBitsConditionChecker : public AbstractThresholdConditionChecker {
 
 public:
     explicit VersionBitsConditionChecker(Consensus::DeploymentPos id_) : id(id_) {}
-    uint32_t Mask(const Consensus::Params& params) const { return ((uint32_t)1) << params.vDeployments[id].bit; }
+    uint32_t Mask(const Consensus::Params& params) const { return ((uint32_t)1) << params.vDeployments.at(id).bit; }
```

Comment:
please convert this to be a c++11 style functional cast

Reply:
I don't think functional casts were added in C++11, pretty sure they existed before already.

Reply:
Yeah, seems like c++11 added the `int{}` syntax but not `int()`. Learn smth new everyday